src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSBE/PartsParam.cs

```diff
@@ -1947,6 +1953,7 @@ public enum HitFilterType : byte
                     Unk17 = 17,
                     Unk19 = 19,
                     Unk20 = 20,
+                    Unk21 = 21,
```

Comment:
Honestly we should have a script to check for incompletenesses like this. Good addition, I'm still malding over enums but that's beyond me to change.